src/pages/contact-list/contact-list.ts

```diff
@@ -17,21 +18,29 @@ export class ContactListPage {
   searchQuery: string = '';
   //For tests
   contacts:any;
+  originalContact: any;
+  token: string;
   verif0Contact: boolean = false;
 
-  constructor(public navCtrl: NavController, public navParams: NavParams, public menuCtrl: MenuController, public contactServices: ContactServicesProvider) {
+  constructor(public navCtrl: NavController, public navParams: NavParams, public menuCtrl: MenuController, public contactServices: ContactServicesProvider, public userServices: UserServicesProvider) {
     this.menuCtrl.enable(true);
 
+    this.token = this.userServices.token
+
   }
 
   ionViewDidLoad() {
-    this.initializeList();
-    this.verif0Contact = (this.contacts.length == 0);
+    this.contactServices.getContacts(this.token).then( contacts =>{
```

Comment:
and here you can call the token with userServices.token without setting a locale variable

Reply:
fixed